typescript/react/best-practice/react-props-spreading.yaml

```diff
@@ -0,0 +1,11 @@
+rules:
+- id: react-props-spreading
+  pattern-either:
```

Comment:
Nit: we don't need the `pattern-either`.

Reply:
thanks! was double checking for stuff like that and still missed this one :smile: